tests/cases/02_connect.ts

```diff
@@ -32,4 +32,15 @@ export default function connectTests() {
     assertEquals(db.name, "my-db");
     client.close();
   });
+
+  Deno.test("test runCommand", async () => {
```

Comment:
`testWithClient` should be used because it is different from the test part of connect

Reply:
:+1: https://github.com/denodrivers/deno_mongo/pull/295/commits/48ca33e3a9cf21de606de8448879830106c91bfb